### DISCUSSION TITLE
Boot from a real physical drive?

### DISCUSSION THREAD

**Discussion by DeDelner** [utmapp/UTM#5065] [UTM Help (macOS)] [answered]
I see it is generally possible in QEMU, just wondering if it is also possible in UTM? If yes, how?

**Comment by Rastafabisch** [comment]
In theory you can create a virtual disk referring to a physical device. therefor you would need to install QEMU via home-brew. Said disk should be able to be imported into UTM. 
This would need to be tested however. For qemu there are plenty of guides online.

**Comment by ktprograms** [comment, marked as answer]
Sandboxed apps (such as UTM) can't access physical drives.

You can either make a drive image as @​Rastafabisch suggested, or share the device to your VM using qemu-nbd (but you can only access it after booting into an OS). Ping me if you'd like instructions on that.

**Reply by aaronfranke** [reply]
@ktprograms Can you provide instructions for these? Either a drive image or qemu-nbd are fine as long as it boots, however I would appreciate having the instructions for both options so that I can try each.

**Comment by Tyler6284** [comment]
Apologies for necroing this, but I just have to ask, and I could be way out of my element.

Is it possible to get UTM 'un-sandboxed' in order to access a physical drive?

**Reply by alisade** [reply]
@ktprograms can you please point me I also need to access physical disk from utm
